src/net/RTP/AudioStream.cs

```diff
@@ -67,10 +68,15 @@ public bool HasAudio
         /// <param name="durationRtpUnits">The duration in RTP timestamp units of the audio sample. This
         /// value is added to the previous RTP timestamp when building the RTP header.</param>
         /// <param name="sample">The audio sample to set as the RTP packet payload.</param>
+        ///
```

Comment:
Could you remove this orphaned summary comment delimiter. It will get flagged as a warning.